content/applications/inventory_and_mrp/inventory/product_management/product_replenishment/reordering_rules.rst

```diff
@@ -93,6 +95,64 @@ For advanced usage of reordering rules, learn about the following reordering rul
 - :ref:`Visibility days <inventory/product_management/visibility-days>`
 - :ref:`Route <inventory/product_management/route>`
 
+0/0/1 reordering rule
+---------------------
+
+The *0/0/1* reordering rule is a specialty rule used to replenish a product that is not kept
+on-hand, each time a sales order (SO) is confirmed for that product.
+
+.. important::
+   The 0/0/1 reordering rule is similar to the *Replenish on Order (MTO)* route, in that both
+   workflows are used to replenish a product upon confirmation of an |SO|.
+
+   The main difference between the two methods is that the Replenish on Order route automatically
+   reserves the product for the SO that caused it to be replenished. This means the product cannot be
+   used for a different |SO|.
+
+   The 0/0/1 reordering rule does not have this limitation. A product replenished using the rule is
+   not reserved for any specific |SO|, and can be used as needed.
+
+   Another key difference is that replenishment orders created by the Replenish on Order route are
+   linked to the original |SO| by a smart button at the top of the order. When using the 0/0/1
+   reordering rule, a replenishment order is created, but is not linked to the original |SO|.
+
+   See the :doc:`Replenish on Order (MTO) <mto>` documentation for a full overview of the MTO route.
+
+To create a 0/0/1 reordering rule, navigate to :menuselection:`Inventory app --> Products -->
+Products`, and select a product.
+
+At the top of the product's page, click the :icon:`fa-refresh` (Reordering Rules) smart button to
+open the reordering rules page for the product. On the page, click :guilabel:`New` to begin
+configuring a new reordering rule.
+
+In the :guilabel:`Route` field of the new reordering rule, select the route the rule should use to
+replenish the item. For example, if the product should be purchased from a vendor, select the
+:guilabel:`Buy` route.
+
+In the :guilabel:`Min Quantity` field and :guilabel:`Max Quantity` field, leave the values set to
+`0.00`. In the :guilabel:`To Order` field, enter a value of `1.00`.
```

Comment:
To my knowledge, we don't manually enter `1 ` in the To Order field. We'd create a sales order so that forecasted quantity is -1, and Odoo automatically increments the To Order value to 1.
```suggestion
`0.00`. With this set up, once an order is placed, the reordering rule is triggered to bring the negative stock value back to 0.
```
Also, feel free to reword this as you see fit! :D

Reply:
I just created a new reordering rule to confirm, and the default To Order value is 0.